vms/administrator/templates/administrator/profile.html

```diff
@@ -5,7 +5,7 @@
 {% block content %}
     <div class="spacer"></div>
         <h1 class="header">
-            {{ administrator.first_name }} {{ administrator.last_name }}
+            <p style = "text-align:center;">{{ administrator.first_name }} {{ administrator.last_name }}</p>
```

Comment:
You can just add some CSS to `.header` instead of adding inline CSS

Reply:
> You can just add some CSS to `.header` instead of adding inline CSS

So should I add the css under ` \vms\vms\static\vms\css` ?